src/hooks/api/organizers.ts

```diff
@@ -48,8 +44,8 @@ const useGetOrganizersByQueryQuery = (
       start: paginationOptions.start,
       limit: paginationOptions.limit,
     },
-    enabled: !!name,
     ...configuration,
+    enabled: !!name && configuration.enabled === true,
```

Comment:
Should this maybe account for when `enabled` is not passed as part of `configuration`? I'm worried it might make `enabled` always false then 🤔

Reply:
This is probably more correct
`!!name && configuration.enabled !== false`

I think we make the same mistake in different parts of the codebase